Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -127,6 +145,23 @@ CFTypeID CGPathGetTypeID() {
 static Boolean __CGPathEqual(CFTypeRef cf1, CFTypeRef cf2) {
     __CGPath* path1 = (__CGPath*)cf1;
     __CGPath* path2 = (__CGPath*)cf2;
+
+    path1->closePath();
```

Comment:
should we be closing the path here, and will doing so change how future operations on the path behave?